os/hal/ports/SN32/LLD/SN32F2xx/USB/usbhw.h

```diff
@@ -59,6 +58,8 @@
 #define mskEP5_DIR                  (0x1<<4)
 #define mskEP6_DIR                  (0x1<<5)
 
+#define mskVREG33DIS_EN             (0x1<<31)
```

Comment:
you're probably referring to 
```suggestion
#define mskVREG33DIS_EN             (0x1<<24)
```
`Enable the VREG33 discharge path to GND.`